components/purchase_modal/icon_message.scss

```diff
@@ -91,5 +79,14 @@
             line-height: 14px;
             text-align: center;
         }
+
+        svg {
+            align-self: center;
+            margin-left: 60px;
+
+            @media screen and (max-width: 600px) {
+                width: 150%;
```

Comment:
I think setting `max-width: 150%` to the element will work as well without needing the media query.